template/javascript/default/src/App.vue

```diff
@@ -9,5 +9,6 @@
 </template>
 
 <script setup>
-  //
+  import HelloWorld from '@/components/HelloWorld.vue'
```

Comment:
This shouldn't be necessary. They are automatically loaded with unplugin-vue-components.